code/modules/projectiles/projectile/energy/net_snare.dm

```diff
@@ -32,13 +32,36 @@
 /obj/effect/nettingportal/Initialize()
 	. = ..()
 	var/obj/item/beacon/teletarget = null
-	for(var/obj/machinery/computer/teleporter/com in GLOB.machines)
-		if(com.target)
-			if(com.power_station && com.power_station.teleporter_hub && com.power_station.engaged)
-				teletarget = com.target
+	
+	for(var/obj/item/beacon/bea in GLOB.teleportbeacons)
+		if(is_eligible(bea) && bea.nettingportal) //is it quick dragnet beacon?
+			teletarget = bea
+
+	// --- THINKING OF REMOVING THIS CODE ENTIRELY
+	if (teletarget==null) //revert to old timer
+		for(var/obj/machinery/computer/teleporter/com in GLOB.machines)
+			if(com.target)
+				if(com.power_station && com.power_station.teleporter_hub && com.power_station.engaged)
+					teletarget = com.target
+	// --- END COMMENT
```

Comment:
I'd agree with the comment, if you are adding new beacons specifically for dragnets then the old stuff shouldn't be used